ts/features/bonus/bonusVacanze/screens/AvailableBonusScreen.tsx

```diff
@@ -81,7 +94,27 @@ class AvailableBonusScreen extends React.PureComponent<Props> {
       <AvailableBonusItem
         bonusItem={item}
         onPress={() =>
-          fromNullable(handlersMap.get(item.id_type)).map(h => h(item))
+          fromNullable(handlersMap.get(item.id_type)).foldL(
```

Comment:
Could leave a comment to explain why the user should update the app when the current code hasn't got a specific handler?